scripts/convert_bias_scan_to_calibration.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import os
+import logging
+import ROOT
+
+#Loads mattak
+ROOT.gSystem.Load('/home/ruben/Documents/software/mattak/install/libmattak.so')
```

Comment:
just load `libmattak.so`, as long as you have the right directory in your LD_LIBRARY_PATH it should work...

Reply:
Good point I forgot to remove my hardcoded path. Thanks for spotting that